Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: build clean test clean
 
 build:
-	dune build
+	dune build @install
```

Comment:
Why this change? `@install` builds strictly less and makes it easy to forget to avoid examples/tests.

Reply:
It was to make it build without requiring core_bench. I am going to send a PR to re-activate it.
We still build all on the ocaml-CI in any case